### PR TITLE
Fix reference of Minerbo1978

### PR DESCRIPTION
**Pull request by hrueter**
## Proposed changes

Add links and a proper journal name to the reference for Minerbo1978.

### Component:

- [ ] Code
- [x] Documentation
- [ ] Build system
- [ ] Continuous integration

### CODE REVIEW

**Review by nilsdeppe — DISMISSED** [review]

**Review by mar-celine — COMMENTED** [review]
Just comparing with the others, should `@ARTICLE` be `@article`?

**Comment by hrueter** [comment]
@gsb76 You are right, but in practice both work.

I also changed the author names to _lastname, firstname_

File: docs/References.bib [hunk 1]
```diff
@@ -308,7 +308,7 @@ @article{Minerbo1978
   year    = "1978",
   issn    = "0022-4073",
   doi     = "10.1016/0022-4073(78)90024-9",
-  url     = "http://www.sciencedirect.com/science/article/pii/0022407378900249"
+  url     = "https://doi.pangaea.de/10.1016/0022-4073(78)90024-9"
```
Comment (nilsdeppe) [inline docs/References.bib]: I think we should go through `doi.org`

Comment (hrueter) [reply docs/References.bib]: I tried to use an alternative that actually guaranties a certain level of information privacy, but I will do as you wish.

Comment (nilsdeppe) [reply docs/References.bib]: If we want to make such I change I think it should be discussed as an issue first and then done to all of them at once :)

**Comment by hrueter** [comment]
I changed the url to doi.org and squashed the minute changes directly.

**Review by nilsdeppe — APPROVED** [review]

**Review by nilsvu — APPROVED** [review]

**Pull request merged**